app/assets/javascripts/services/preferencesManager.ts

```diff
@@ -25,6 +25,9 @@ export class PreferencesManager extends ApplicationService {
   }
 
   streamPreferences() {
+    if (!this.userPreferences) {
```

Comment:
Hmm..it looks like `this.userPreferences` will always be null at this point. In which case the solution should be:

1. Rename `loadSingleton` to `reloadSingleton` to indicate it can be called more than once.
2. Call `reloadSingleton` in `onAppLaunch` above `streamPreferences`

Reply:
Done :+1: